homeassistant/components/remote/harmony.py

```diff
@@ -215,10 +218,14 @@ def send_command(self, commands, **kwargs):
 
         num_repeats = kwargs.get(ATTR_NUM_REPEATS)
         delay_secs = kwargs.get(ATTR_DELAY_SECS, self._delay_secs)
+        hold_secs = kwargs.get(ATTR_HOLD_SECS)
+        _LOGGER.debug("Sending commands to device %s holding for %s seconds",
+                      device, hold_secs)
 
         for _ in range(num_repeats):
             for command in commands:
-                self._client.send_command(device, command)
+                _LOGGER.debug("Sending command %s", command)
+                self._client.send_command(device, command, hold_secs)
                 time.sleep(delay_secs)
```

Comment:
This should really be removed too.

Reply:
@MartinHjelmare, what is the solution then to have something wait a certain amount of time and not having to rewrite something existing completely to async for a small change?